server/models/blogs.js

```diff
@@ -0,0 +1,34 @@
+import mongoose from 'mongoose';
+
+const { Schema } = mongoose;
+
+const BlogsSchema = new Schema({
+  userId: {
+    type: Schema.Types.ObjectId,
+    ref: 'User',
+  },
+  title: {
+    type: String,
+    required: true,
+  },
+  content: [
+    {
+      _id: Schema.Types.ObjectId,
+      type: String,
+      contentFragment: {
```

Comment:
Why not separate collection and then joined by tef?

Reply:
[The same reasoning as before](https://github.com/pesto-students/batch-9-Lumen/pull/43#discussion_r314897577)
Thoughts?

Reply:
You can just do a `fragment` instead of `contentFragment`